modules/gatsby/src/gatsby-config/get-gatsby-config.js

```diff
@@ -118,6 +118,15 @@ module.exports = function getGatsbyConfig(config) {
         }
       },
 
+      // Handles MDX files
+      {
+        resolve: 'gatsby-mdx',
+        options: {
+          extensions: ['.mdx']
+        }
+      },
```

Comment:
just want to point out that this is precisely where ocular-gatsby is useful. existing sites just have to upgrade ocular-gatsby, not change their manual configurations or add new dependencies :)